katalog/dex/web/templates/header.html

```diff
@@ -13,14 +13,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="{{ url .ReqPath "static/main.css" }}" rel="stylesheet">
     <link href="{{ url .ReqPath "theme/styles.css" }}" rel="stylesheet">
-    <link rel="icon" href="{{ url .ReqPath "theme/favicon.png" }}">
+    <link rel="icon" href="{{ url .ReqPath "theme/favicon.ico" }}">
   </head>
 
   <body class="theme-body">
     <div class="theme-navbar">
       <div class="theme-navbar__logo-wrap">
         <img class="theme-navbar__logo" src="{{ url .ReqPath logo }}">
-        <h1>Fury Login</h1>
+        <h1>SKD Login</h1>
```

Comment:
let's use the full name here
```suggestion
        <h1>SIGHUP Kubernetes Distribution Login</h1>
```